assets/javascripts/needleeditor.js

```diff
@@ -348,7 +365,23 @@ function loadTagsAndName() {
   $("#needleeditor_tags").find('input').each(function() {
     $(this).prop('checked', tags.indexOf($(this).val()) !== -1);
   });
-  $("#property_workaround").prop('checked', $.inArray('workaround', needle.properties) !== -1);
+	
+	var workaround_flag = 0;
+	for (var i = 0; i < needle.properties.length; i++){
+		if (needle.properties[i].name == "workaround"){
```

Comment:
In general, `===` is preferred over `==` in JavaScript. Same counts for the other lines where you use `==` unless the possible type conversion is wanted.

Reply:
I will fix it